Makefile

```diff
@@ -40,16 +40,26 @@ verify:
 	    exit 1; \
 	fi
 
+release:
```

Comment:
Question: How does this compare to releasing other tools?

Reply:
the idea for this project is to have the release binaries ready to be downloaded. all of our products are released via container images, but this one is meant to be used as a command line tool